src/main/resources/config.yml

```diff
@@ -1,2 +1,18 @@
-markerFile: markers.json
-interval: 5
\ No newline at end of file
+# Backend type, supported: MySQL
+backend: "MySql"
+backendConfig:
+  # MySQL connection url
```

Comment:
Maybe it is better to simplify config and left only connection string + credentials?